### DISCUSSION TITLE
failed to create mdnotes file

### DISCUSSION THREAD

**Discussion by Lulu-Lime** [argenos/zotero-mdnotes#227] [Q&A]
Recently when I click the menu "Create Mdnotes file" ,there is no responding. But the other two menu works. I tried reinstalling but it didn't work.

Zotero 6.0.27  & Mdnotes for Zotero 0.2.3
Any help would be appreciated.

**Comment by lunawbr** [comment]
I met the same problem and found the solution to this! (https://github.com/argenos/zotero-mdnotes/issues/226)
Edit the MDnotes preferences and untick the box "Use the item's citekey as title?", then it works!

**Reply by Yejing-Zhan** [reply]
It works, amazing. Thank you

**Comment by hyh712** [comment]
我取消了但是依旧无法解决，使用mdnotes无法生成md文件并保存
